apps/modeler/include/pcl/apps/modeler/main_window.h

```diff
@@ -128,8 +128,7 @@ namespace pcl
         friend class AbstractItem;
 
         MainWindow();
-        MainWindow(const MainWindow &) {} // copy ctor hidden
```

Comment:
This guy probably should also be marked `= delete`. The copy ctor and copy assignment are independent.

Reply:
I found the following statement on Wikipedia:

>     If a destructor is declared generation of a copy constructor is deprecated (C++11, proposal N3242 [2]).

If it's correct, then no copy constructor will be generated?

Reply:
I now tried to dig into another source
https://en.cppreference.com/w/cpp/language/copy_constructor

It states the same as you quoted from wiki. 

I interpret the same as you. Worst case scenario we can test it on cpp-shell.